Source/Core/Common/GL/GLInterface/GLInterface.cpp

```diff
@@ -20,8 +20,6 @@
 #include "Common/GL/GLInterface/EGL.h"
 #elif ANDROID
 #include "Common/GL/GLInterface/EGLAndroid.h"
-#else
-#error Platform doesnt have a GLInterface
```

Comment:
Probably what should happen is that the `SKIP_GL`/`HAVE_GL` flags omit this subdirectory altogether.